src/permissions.rs

```diff
@@ -60,7 +64,7 @@ impl DenoPermissions {
     };
     // TODO get location (where access occurred)
     let r = permission_prompt(&format!(
-      "Deno requests network access to \"{}\".",
+      "⚠️  Deno requests network access to \"{}\".",
```

Comment:
Can you "DRY" this up so that the emoji is only listed once? EG
```rust
permission_prompt(format!("network access to \"{}\", domain_name))
```

Reply:
@ry fixed